test/gpdb/run_docker.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -ex 
+
+DOCKERIMAGE="$1"
+DOCKER_GPDB_SRC_PATH="${2:-/home/gpadmin/gpdb_src}"
+LOG_DIR="${3:-$(pwd)/logs}"
+DOCKER_GPDB_SRC_PARENT=$(dirname "$DOCKER_GPDB_SRC_PATH")
+SCRIPT_PATH=$(dirname $(realpath -s $0))
+PROJECT_ROOT=$(dirname $(dirname "$SCRIPT_PATH"))
+
+if [ ! -d "$LOG_DIR" ]; then
+	echo "Directory $LOG_DIR does not exist. Creating now."
+	mkdir -p "$LOG_DIR"
+fi
+
+# we need to restore the repo ownership after docker is finished
+# otherwise we won't be able to access anything here, because
+# this repo becomes a volume in docker run
+fileOwnerUid=$(stat -c '%u' .)
+fileOwnerGid=$(stat -c '%g' .)
```

Comment:
The issue I'am trying to solve is that docker changes the ownership of the whole repo.
Probably it happens due to `-v $PROJECT_ROOT:$DOCKER_GPDB_SRC_PARENT/pgbackres` launch. I'll investigate this problem a little bit more